rest-tests/src/test/java/io/syndesis/qe/rest/tests/integrations/ConnectionsGeneralSteps.java

```diff
@@ -216,4 +219,32 @@ public void createHTTPConnection() {
         log.info("Creating HTTP connection {}", httpConnection.getName());
         connectionsEndpoint.create(httpConnection);
     }
+
+    @Given("^connection \"([^\"]*)\" is created$")
```

Comment:
use imperative

Reply:
this is questionable. I've read article about not using imperative in gherkin at all. Is there any specific reason to use it ?

Reply:
everything else (or probably more than 95%) is in imperative